### PR TITLE
Add job that executes shoot comparison

### PR DESCRIPTION
**Pull request by akgalwas**
**Description**

Changes proposed in this pull request:

- Yaml files prepared
- Simple helper scripts implemented

**Related issue(s)**
#185

### CODE REVIEW

File: hack/shoot-comparator/README.md [hunk 1]
```diff
@@ -17,4 +17,87 @@ CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -o ./bin/shoot-comparator ./cmd
...
+- garden-kyma-dev-kim-pvc-test3.yaml
+- garden-kyma-dev-kims-aws-pvc-shootCR.yaml
+------------------------------------------------------------------------------------------
+
+------------------------------------------------------------------------------------------
+Files existing in /Users/i326211/dev/temp/kim-test/shoot-comparator/test3/provisioner folder only:
+- garden-kyma-dev-a2f8da9.yaml
+- garden-kyma-dev-a3008c5.yaml
+------------------------------------------------------------------------------------------
+```
```
Comment (koala7659) [inline hack/shoot-comparator/README.md:103]: You can add one or two sentences with information that all created resources are left on the cluster and the user should delete them manually after finishing comparison process.

**Review by koala7659 — DISMISSED** [review]
LGTM - small remark - please check my comment

**Review by m00g3n — APPROVED** [review]
LGTM

**Review by koala7659 — DISMISSED** [review]
LGTM

File: hack/shoot-comparator/scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+In such case you can check the pod status manually by executing the following commands:
+```bash
+kubectl get po fetch-test-comparison-results -n kcp-system
+kubectl describe po fetch-test-comparison-results -n kcp-system
+```
+
+## Comparing files starting from a specific date
+
+If you want to compare filed older than a specific date, you can specify the date in the `./manifests/job.yaml` script.
```
Comment (grego952) [inline hack/shoot-comparator/scripts/README.md:102]:
```suggestion
If you want to compare files older than a specific date, you can specify the date in the `./manifests/job.yaml` script. 
```

File: hack/shoot-comparator/scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+```bash
+kubectl get po fetch-test-comparison-results -n kcp-system
+kubectl describe po fetch-test-comparison-results -n kcp-system
+```
+
+## Comparing files starting from a specific date
+
+If you want to compare filed older than a specific date, you can specify the date in the `./manifests/job.yaml` script. 
+
+## Analysing comparison results
```
Comment (grego952) [inline hack/shoot-comparator/scripts/README.md:104]:
```suggestion
## Analyzing comparison results
```

File: hack/shoot-comparator/scripts/README.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+If any differences were detected you can analyse the results by examining the content of the `result.txt` file stored in the output directory. The file will contain the details of the comparison, such as the names of the files that differ.
```
Comment (grego952) [inline hack/shoot-comparator/scripts/README.md:106]:
```suggestion
If any differences were detected you can analyze the results by examining the content of the `result.txt` file stored in the output directory. The file will contain the details of the comparison, such as the names of the files that differ.
```

**Review by m00g3n — APPROVED** [review]
LGTM

**Review by grego952 — APPROVED** [review]

**Pull request merged**